Formula/ghc@8.10.rb

```diff
@@ -0,0 +1,127 @@
+class GhcAT810 < Formula
+  desc "Glorious Glasgow Haskell Compilation System"
+  homepage "https://haskell.org/ghc/"
+  url "https://downloads.haskell.org/~ghc/8.10.7/ghc-8.10.7-src.tar.xz"
+  sha256 "e3eef6229ce9908dfe1ea41436befb0455fefb1932559e860ad4c606b0d03c9d"
```

Comment:
Just went with 8.10 as it was previously working version and should be easier to downgrade if necessary. Might be able to use 9.0 instead with some extra work (e.g. newer Stack resolvers, etc).